compatibility-verifier/sample-test-suite/post-server-rollback.yaml

```diff
@@ -44,3 +58,14 @@ operations:
     description: Run query on FeatureTest2 using SQL
     queryFileName: queries/feature-test-2-sql-realtime.queries
     expectedResultsFileName: query-results/feature-test-2-sql-realtime.results
+  - type: queryOp
+    description: Run query on FeatureTest3 using SQL
+    queryFileName: queries/feature-test-3-sql-realtime.queries
+    expectedResultsFileName: query-results/feature-test-3-sql-realtime.results
+    shouldCompareNumEntriesScannedInFilter: true
+    shouldCompareNumConsumingSegmentsQueried: true
```

Comment:
Why do we need this?

Reply:
Removed, as we discussed offline, these comparisons should fire automatically if these fields exit in metadata of query results. Also removed the corresponding fields in existing query comparisons as we don't check them.